OpenGov-OpenData/ckanext-orgportals
# [OD-944] Add dataset button missing from an organization's page w/ orgportals enabled

## [[OD-944] Add dataset button missing from an organization's page w/ orgportals enabled](https://opengovinc.atlassian.net/browse/OD-944)

## Description
When the orgportals extension is enabled the `Add Dataset` button disappears from an organization page. The problem is found in the organization read template.

The read template in this extension was previously added to change the pagination bar. However, the template block `page_primary_action` is missing causing the `Add Dataset` button to disappear.

The pagination change is not needed so the organization read template is being deleted.

## Testing
1) Install the orgportal extension. Use the `og-org-portal` branch.
2) Start CKAN
3) Go to an organization page and check that the  `Add Dataset` button is missing
5) Checkout the PR branch
6) Restart CKAN
7) Go to an organization page and check that the  `Add Dataset` button is present now
